### PR TITLE
Add commit message pratices

### PR DESCRIPTION
**Pull request by garaujodev**
Improve our guidelines, inserting commit messages pratices.

The discussion is open.

### CODE REVIEW

File: resources/commit_messages.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Commit Messages
+=======
+
+This pratices are based on [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/) article, by Chris Beams
```
Comment (wevtimoteo) [inline resources/commit_messages.md]:
```suggestion
These practices are based on [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/) article, by Chris Beams.
```

File: resources/commit_messages.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Commit Messages
+=======
```
Comment (wevtimoteo) [inline resources/commit_messages.md]: A valid title in markdown should contain the size of characteres for `=`, for example:

```md
Commit Messages
===============
```

Or

```
# Commit Messages
```

File: resources/commit_messages.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+This pratices are based on [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/) article, by Chris Beams
+
+## Commit Messages
+
+  * Separate subject from body with a blank line
```
Comment (wevtimoteo) [inline resources/commit_messages.md]: You don't need to indent this list, you can just simply:

```md
* Item 1
* Item 2
```

**Review by wevtimoteo — APPROVED** [review]
Consider my comments then :rocket:

**Review by beatrizChagas — APPROVED** [review]
I agree with @wevtimoteo's comments.

File: resources/commit_messages.md (hunk 1 above)
Comment (garaujodev) [reply resources/commit_messages.md]: Fixed in 83f2a53

File: resources/commit_messages.md (hunk 2 above)
Comment (garaujodev) [reply resources/commit_messages.md]: Fixed in 83f2a53

File: resources/commit_messages.md (hunk 3 above)
Comment (garaujodev) [reply resources/commit_messages.md]: fixed in 83f2a53

**Comment by garaujodev** [comment]
Thanks for the comments.

**Pull request merged**